### PR TITLE
Feature 56: Bug when create sick hours response message comes back as nonbillable

### PR DESCRIPTION
**Pull request by DCisero**
https://integralsoftwareinc.visualstudio.com/IntegralTimeTracker/_workitems/edit/56/

-Fixed bug to include sick hours in response message 
-Fixed bug to include vacation hours in response message

### CODE REVIEW

File: src/TimeTracker.Library/Services/Interpretation/ReportInterpreter.cs [hunk 1]
```diff
@@ -7,12 +7,22 @@
 
 namespace TimeTracker.Library.Services.Interpretation
 {
+    public enum ReportType
```
Comment (ambercs) [inline src/TimeTracker.Library/Services/Interpretation/ReportInterpreter.cs]: thanks for adding this enum to make parsing clearer [JR]

File: src/TimeTracker.Library/Services/Orchestration/RecordMessageOrchestration.cs [hunk 2]
```diff
@@ -40,6 +41,16 @@ protected override async Task<SlackMessageResponse> RespondTo(HoursInterpretedMe
                     $"Registered *{message.Hours:F1} hours* for project *{message.Project}* {message.Date:D}. " +
                     (message.IsWorkFromHome ? "_Worked From Home_" : ""), true);
             }
+            
+            if (message.TimeEntryType == TimeEntryTypeEnum.Sick)
```
Comment (ambercs) [inline src/TimeTracker.Library/Services/Orchestration/RecordMessageOrchestration.cs]: 👍 [JR]

File: test/TimeTracker.Library.Test/Services/Orchestration/RecordTests.cs [hunk 3]
```diff
@@ -73,6 +73,28 @@ public async Task HandleCommand_hours_processRecordOption_shouldFailIfNotEntirel
             database.TimeEntries.Should().BeEmpty();
             slackMessage.Text.Should().Be($"Error: *Not sure how to interpret 'some nonsense'*");
         }
+        
+        [Fact]
+        public async Task WhenRecordSickHours_SlackMessageIncludesSickHoursWereRecorded()
```
Comment (ambercs) [inline test/TimeTracker.Library.Test/Services/Orchestration/RecordTests.cs]: good to see we have a test for proving the bug [JR]

File: src/TimeTracker.Library/Services/Interpretation/ReportInterpreter.cs (hunk 1 above)
Comment (DCisero) [reply src/TimeTracker.Library/Services/Interpretation/ReportInterpreter.cs]: Nate's great idea!

File: src/TimeTracker.Library/Services/Orchestration/RecordMessageOrchestration.cs [hunk 4]
```diff
@@ -43,22 +43,31 @@ protected override async Task<SlackMessageResponse> RespondTo(HoursInterpretedMe
                     (message.IsWorkFromHome ? "_Worked From Home_" : ""), true);
             }
             
-            if (message.TimeEntryType == TimeEntryTypeEnum.Sick)
+            switch (message.TimeEntryType)
```
Comment (JamesReate) [inline src/TimeTracker.Library/Services/Orchestration/RecordMessageOrchestration.cs]: good to change for a switch stmt!

**Pull request merged**